airbyte-webapp/src/types/formik.ts

```diff
@@ -0,0 +1,3 @@
+import { FormikConfig } from "formik";
+
+export type FormikOnSubmit<T> = FormikConfig<T>["onSubmit"];
```

Comment:
I'd be tempted to just put this into `TransformationView.tsx` since that's the only place it's used, just to cut down on files. 🤷

Reply:
This is pretty generic and may be needed later. I rather make it available to all now than to hide it inside a component.